source/ast/builtins/SystemTasks.cpp

```diff
@@ -725,9 +725,9 @@ class SdfAnnotateTask : public SystemTaskBase {
     }
 };
 
-class PlaTask : public SystemTaskBase {
+class SLANG_EXPORT PlaTask : public SystemTaskBase {
```

Comment:
Adding SLANG_EXPORT here doesn't do anything; this class is not externally visible to downstream users of the library.

Reply:
removed